conf/squid/common/13node-4client-single-site-regression.yaml

```diff
@@ -0,0 +1,120 @@
+# Single site cluster with 13 Ceph & 4 Client nodes
+# with daemons - 13 crash, 5 mon, 3 mgr, 3 rgw
+# 5 nfs, 10 (+2 backup) osds and
+# 1 alertmanager, node-exporter, prometheus, grafana
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+        role:
+          - _admin
+          - installer
+          - crash
+          - alertmanager
+          - mon
+      node2:
+        role:
+          - crash
+          - mon
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node3:
+        role:
+          - crash
+          - mon
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node4:
+        role:
+          - crash
+          - rgw
+          - mgr
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node5:
+        role:
+          - crash
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node6:
+        role:
+          - crash
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node7:
+        role:
+          - crash
+          - node-exporter
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node8:
+        role:
+          - crash
+          - prometheus
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node9:
+        role:
+          - crash
+          - grafana
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node10:
+        role:
+          - crash
+          - rgw
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node11:
+        role:
+          - crash
+          - rgw
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node12:
+        role:
+          - crash
+          - mon
+          - nfs
+          - osd-bak
+        no-of-volumes: 4
+        disk-size: 20
+      node13:
+        role:
+          - crash
+          - mon
+          - mgr
+          - osd-bak
+        no-of-volumes: 4
+        disk-size: 20
+      node14:
+        role:
+          - client
+      node15:
+        role:
+          - client
+      node16:
+        role:
+          - client
+      node17:
+        role:
+          - client
```

Comment:
@vamahaja 
The requirement for RADOS CIDR blocklisting test:
- Total number of clients: 4
- Two clients part of one subnet, other 2 part of another subnet

Can this request be accommodated? If not, then we need should make a note and not run CIDR blocklisting test suite with this cluster conf

Reply:
This is doable with current support, as we are doing it for stretch cluster wiht opestack enviroment. then blocklist on NW CIDR can be called dynamically without having issues.
`conf/squid/rados/7node-1client-stretch-mode-rhos-d.yaml`